Gruntfile.js

```diff
@@ -44,15 +44,37 @@ module.exports = function (grunt) {
                 }
             }
         },
+        clean: {
+            previousBuild: {
+                src: ["dist/"]
+            }
+        },
+        copy: {
+            infusion: {
+                expand: true,
+                flatten: true,
+                src: "node_modules/infusion/dist/infusion-all.min.js",
+                dest: "dist/js/lib/infusion/"
+            },
+            source: {
+                expand: true,
+                cwd: "src",
+                src: "**",
+                dest: "dist/"
+            }
+        },
         licenseBanner: grunt.file.read("templates/LICENSE-banner.txt"),
```

Comment:
I thought you had moved these variables in the banner plugin's section in another pull?

Reply:
As I could recall, I thought we had to move these variables below the task definitions. So I added these new task definitions below the banner task and above the convenience variables.

Perhaps I don't understand if you mean to move the variables somewhere else. Could you please clarify on the same?

Reply:
@the-t-in-rtf, I tried moving these variables under the `useBanner` block, but sadly, it shows some warnings when used. I think it interferes with the `useBanner`'s configuration. So the best thing we can do is to keep these definitions alongside (either above or below) the `useBanner` task.

Reply:
I'm fine with that.